autogluon/utils/tabular/metrics/classification_metrics.py

```diff
@@ -254,3 +257,49 @@ def prior_log_loss(frac_pos, task):
     score = (pac - base_pac) / np.maximum(eps, (1 - base_pac))
 
     return score
+
+
+def confusion_matrix(solution, prediction, labels=None, weights=None):
+    y_type, solution, prediction = _check_targets(solution, prediction)
+    if solution.size == 0 or prediction.size == 0:
+        return np.zeros((n_labels, n_labels), dtype=np.int)
```

Comment:
has `n_labels` been set at this point? Wouldn't we want to raise an Exception here?

Reply:
Should add a unit test for this as well